browser/themes/brave_theme_service_browsertest.cc

```diff
@@ -33,7 +37,56 @@
 #include "base/win/registry.h"
 #endif
 
-using BraveThemeServiceTest = InProcessBrowserTest;
+#if BUILDFLAG(ENABLE_EXTENSIONS)
+#include "chrome/browser/extensions/crx_installer.h"
+#include "chrome/browser/extensions/extension_install_prompt.h"
+#include "chrome/browser/extensions/extension_service.h"
+#include "chrome/common/extensions/extension_constants.h"
+#include "components/crx_file/crx_verifier.h"
+#include "extensions/browser/crx_file_info.h"
+#include "extensions/browser/extension_dialog_auto_confirm.h"
+#include "extensions/browser/extension_system.h"
+#include "extensions/browser/test_extension_registry_observer.h"
+#endif
+
+class BraveThemeServiceTest : public InProcessBrowserTest {
+ public:
+  BraveThemeServiceTest() { brave::RegisterPathProvider(); }
+
+  bool UsingCustomTheme(const ThemeService& theme_service) {
+    return !theme_service.UsingSystemTheme() &&
+           !theme_service.UsingDefaultTheme();
+  }
+
+  base::FilePath GetTestDataDir() {
```

Comment:
This seems not to be used when `BUILDFLAG(ENABLE_EXTENSIONS)` like InstallExtension() below.

Reply:
Moved all new methods into extensions buildflag.